howso/engine/trainee.py

```diff
@@ -3992,7 +3998,10 @@ def delete_trainee(
 
 def load_trainee(
     file_path: PathLike,
-    client: t.Optional[AbstractHowsoClient] = None
+    client: t.Optional[AbstractHowsoClient] = None,
+    *,
+    persistence: Persistence = 'allow',
+    runtime: t.Optional[TraineeRuntimeOptions] = None
```

Comment:
Do we need to include runtime here? Looks like its never used.

Reply:
No, this is stale from adapting a previous PR.  I'll remove it.